week2/scrambleeggs.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <title>Scramble Eggs</title>
+  </head>
+  <body>
+    <script>
+    let texttotest = '';
```

Comment:
Create two inputs to retrieve the values from. Make sure to print whether it's success or not on the webpage (not console).